easybuild/tools/parallelbuild.py

```diff
@@ -131,13 +133,13 @@ def submit_jobs(ordered_ecs, cmd_line_opts, testing=False):
     # compose string with command line options, properly quoted and with '%' characters escaped
     opts_str = subprocess.list2cmdline(opts).replace('%', '%%')
 
-    command = "unset TMPDIR && cd %s && eb %%(spec)s %s --testoutput=%%(output_dir)s" % (curdir, opts_str)
+    command = "unset TMPDIR && cd %s && eb %%(spec)s %s %%(add_opts)s--testoutput=%%(output_dir)s" % (curdir, opts_str)
```

Comment:
I'm not sure the `unset $TMPDIR &&` is still needed (since `eb` set it itself), but it was at some point...

Tried to figure it out with `git log -L136,136:easybuild/tools/parallelbuild.py`, but no luck, it's been there since forever.

Probably since before EB starting setting `$TMPDIR` and taking control over what directory `tempfile.*` is using, so it's likely it can go. But I'm not going to do that in this PR. :-)